### DISCUSSION TITLE
How to re-enable a SBIE notification I disabled?

### DISCUSSION THREAD

**Discussion by w311ang** [sandboxie-plus/Sandboxie#2175] [Q&A Feedback]
I checked global config and sandbox config. I can't see the SBIE code I disabled.

**Comment by bastik-1001** [comment]
You can reset **all** hidden messages with the GUI of the classic version and the plus version.

For the classic version that should be "Configure > Forget Hidden Messages" and for the plus version it is "Options > Reset all hidden messages".

Of course, that is not what you asked, but I failed to see how individual messages can be re-enabled.

**Reply by DavidXanatos** [reply]
you can re enable them one by one by editing the ini files
